ansible/roles/cni/tasks/cilium.yml

```diff
@@ -0,0 +1,61 @@
+---
+- name: Install Helm v3
+  shell: |
```

Comment:
i'd like to avoid using the shell module if possible. as far as i am aware, we haven't _yet_ tainted the sanctity of this repo with use of the shell module. 

i don't know how comfortable i am with running arbitrary bash scripts against my own cluster, much less other people.

of course, since this from the official source(s) of this particular script that makes it slightly more ok. sometimes there is no getting around it; occasionally it's unavoidable. i'd be okay approving this so long as we make an effort to try and get around it somehow if we can. :)

Reply:
Yup for sure. And I had chatted with @xUnholy about this as well in Discord. The installation of Cilium was much more elegant with Helm than with the manifest files you could template out and install. It likely can be done, but it's going to be work.